deltachat-ios/Controller/SettingsController.swift

```diff
@@ -188,6 +189,13 @@ internal final class SettingsViewController: UITableViewController, ProgressAler
         return cell
     }()
 
+    private lazy var qrCodeBackupCell: ActionCell = {
+        let cell = ActionCell()
+        cell.tag = CellTags.qrCodeBackup.rawValue
+        cell.actionTitle = String.localized("transfer_backup")
```

Comment:
i would call this "Set Up New Device" or "Set Up Another Device".

i would not use the wording "Backup" in the UI, this is an implementation details that does not really matter the the user, also, the expectation of a backup is usually sth. else.